<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FCS_IoT Patch Tool v.1.0</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>701</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/100.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Arial Rounded MT Bold&quot;;</string>
   </property>
   <property name="text">
    <string>Select Patch Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Please select the type of patch you would like to perform.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPalette textPalette;
textPalette.setBrush( QPalette::WindowText, QBrush( QColor(211, 211, 211 )));
m_pTextLabel-&gt;setPalette( textPalette );</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#919191;&quot;&gt;KITRI BOB 9th, FCS_IoT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>123</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>123</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Patch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>123</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Patch for:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_adb">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>108</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ADB</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_pm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>108</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PM</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_both">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>231</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Both (recommended)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
